lib/trailblazer/operation/public_call.rb

```diff
@@ -46,13 +61,13 @@ def options_for_public_call(*args)
 
     # Compile a Context object to be passed into the Activity::call.
     # @private
-    def self.options_for_public_call(options, **flow_options)
+    def self.options_for_public_call(options, flow_options = {})
       Trailblazer::Context(options, {}, flow_options[:context_options])
     end
 
     # @semi=public
-    def flow_options
```

Comment:
Why are we not calling it `flow_options` like in the original version? I find "original_..." a bit counter-intuitive, maybe `canonical_flow_options` or `default_flow_options`? Or `#flow_options` :joy:

Reply:
I wanted the method name to be something different than the argument name `flow_options` 😅 .
`default_flow_options` is also cool, will update.

Reply:
How about naming it `flow_options_for_public_call ` ? Its similar to `options_for_public_call ` for building `ctx`.

Reply:
:+1: That's a cool idea! :heart:

Reply:
Updated it!